torch/utils/_sympy/functions.py

```diff
@@ -267,8 +267,11 @@ def eval(
             terms = []
             for term in sympy.Add.make_args(base):
                 quotient = term / divisor
-
-                if quotient.is_integer:
+                # sympy can generate a quotient with (1/22)*.... such that quotient.is_integer is True
```

Comment:
Add https://github.com/sympy/sympy/pull/28442 here?

Reply:
Let's check for `sympy.__version__` and apply this workaround only if the version is less than `1.14.0`

Reply:
is this just a problem for Mul @isuruf